formula_renames.json

```diff
@@ -16,6 +16,7 @@
   "hamsterdb": "upscaledb",
   "heroku-toolbelt": "heroku",
   "kotlin-compiler": "kotlin",
+  "juju": "juju@2.0",
```

Comment:
It looks like the rename was to `juju@1.25`. Also, Please add an alias for `juju` to `juju@2.0`.